### DISCUSSION TITLE
Entryfying globals: does querying in multiple templates affects resource usage/page speed?

### DISCUSSION THREAD

**Discussion by ggallo909** [craftcms/cms#15960] [General]
How do you manage getting the info from single entries in multiple templates (eg: footer.twig + header.twig) without performing multiple queries? Does Craft re-use the same query result across multiple templates or does every query count if done in multiple templates? With Globals the query is done once and injected in the view, but with entries?

**Comment by thupsi** [comment]
I think you don't have to worry about, if I understand correctly: https://github.com/craftcms/cms/pull/12787

**Reply by ggallo909** [reply]
Thank you! I completely missed this
